lib/trainman/build.js

```diff
@@ -1,6 +1,9 @@
 'use strict';
 
 var path = require( 'path' );
+var print = require('gulp-print');
+var filter = require('gulp-filter');
+var merge = require('merge-stream');
```

Comment:
We don't need these because everything is already loaded with gulp-load-plugins. Just use plugins.pluginName. Also, print can be removed. And I don't think we need merge - I think we can use streamqueue. But if we do need it, it should be added to the package dependencies.